entries/t/ticketsource.co.uk.json

```
@@ -0,0 +1,15 @@
+{
+  "TicketSource": {
+    "domain": "ticketsource.co.uk",

```

```suggestion
    "domain": "ticketsource.co.uk",
    "additional-domains": [
      "ticketsource.eu",
      "ticketsource.us"
    ]
```

They also have EU and U.S. sites, which share a common authentication system